.drone.star

```diff
@@ -3088,30 +3014,20 @@ def uploadTracingResult(ctx):
         },
     }]
 
-def publishTracingResult(ctx, suite):
+def logTracingResult(ctx, suite):
     status = ["failure"]
+
     if ("with-tracing" in ctx.build.title.lower()):
         status = ["failure", "success"]
 
     return [{
-        "name": "publish-tracing-result",
+        "name": "log-tracing-result",
         "image": OC_UBUNTU,
         "commands": [
             "cd %s/reports/e2e/playwright/tracing/" % dir["web"],
-            'echo "<details><summary>:boom: To see the trace, please open the link in the console ...</summary>\\n\\n<p>\\n\\n" >> %s/comments.file' % dir["web"],
-            'for f in *.zip; do echo "#### npx playwright show-trace $CACHE_ENDPOINT/$CACHE_BUCKET/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/tracing/$f \n" >> %s/comments.file; done' % dir["web"],
-            'echo "\n</p></details>" >> %s/comments.file' % dir["web"],
-            "more %s/comments.file" % dir["web"],
+            'echo "To see the trace, please open the following link in the console"',
+            'for f in *.zip; do echo "#### npx playwright show-trace  https://cache.owncloud.com/public/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/tracing/$f \n"; done',
         ],
-        "environment": {
-            "TEST_CONTEXT": suite,
-            "CACHE_ENDPOINT": {
-                "from_secret": "cache_s3_server",
-            },
-            "CACHE_BUCKET": {
-                "from_secret": "cache_s3_bucket",
-            },
-        },
```

Comment:
I have removed the secrets and hardcoded it in the link as it should be like `https://cache.owncloud.com/public/..`

Reply:
`https://cache.owncloud.com/public` is supposed to be public anyway, so there should not be any problem exposing this by hardcoding it here.